### ISSUE TITLE
Plotting cohen's kappa correlation matrix

### ISSUE THREAD

**Issue by SnBuenafe** [tinbuenafe/spatialplanr_Collab#5]
- create the matrix and plot it
- inputs would be the solutions (sf?)

**Comment by AlviDab** [comment]
If you want I have a function based on your Cohen's Kappa function that produce the corrplot with a list of solutions in input

**Comment by SnBuenafe** [comment]
I edited and pulled the function into the `main` branch. It works for me. Could you please check if it works for you @AlviDab Thanks.

Once it works for you, I think we can submit a PR to the SpatialPlanningR repo of MME.

**Comment by AlviDab** [comment]
It works for me, I just deleted a comment that was in an old version of the function.

**Comment by SnBuenafe** [comment]
Thanks. Just so we're both in the same page: when we're making functions, we should be making them in separate branches (which I realized now I shouldn't have pulled into the main). So we'll keep the branches open until we've successfully made a pull request to the SpatialPlanningR repo of MME.

After this comment, I'll create a new branch (fxn_cohen).

For other functions, let's probably do this formatting:
`fxn_circularbarplot` : indicates the branch where we'll both pull into or where we'll branch off to our separate branches
`fxn_circularbarplot_tin` : indicates the branch that I branched off from `fxn_circularbarplot` and where i'll be working independently
`fxn_circularbarplot_alvise` : indicates the branch that you branched off from `fxn_circularbarplot` and where you'll be working independently

Then once we've convened, we can pull into `fxn_circularbarplot` and that's where we'll make the pull request into SpatialPlanningR. Let me know if you have any other suggestions or if it's good for you.
